### PR TITLE
Code style cleanup

### PR DESCRIPTION
**Pull request by danielmarbach**
Connects to #67

Replaces https://github.com/Particular/NServiceBus.AzureStorageQueues/pull/78

### CODE REVIEW

**Comment by danielmarbach** [comment]
@Particular/azure-maintainers can you review and merge if green?

File: src/Transport/Config/DefaultConfigurationValues.cs [hunk 1]
```diff
@@ -28,5 +20,13 @@ void ApplyDefaults(SettingsHolder settings)
             settings.SetDefault(WellKnownConfigurationKeys.PurgeOnStartup, DefaultPurgeOnStartup);
             settings.SetDefault(WellKnownConfigurationKeys.DefaultQueuePerInstance, DefaultQueuePerInstance);
         }
+
+        const int DefaultMessageInvisibleTime = 30000;
```
Comment (melkio) [inline src/Transport/Config/DefaultConfigurationValues.cs]: Why aren't these settings inlined in the `SetDefault` call? For me setup should be more readable

File: src/Transport/ConnectionString.cs [hunk 2]
```diff
@@ -48,12 +46,14 @@ public override bool Equals(object obj)
...
-            return Equals((ConnectionString)obj);
+            return Equals((ConnectionString) obj);
         }
 
         public override int GetHashCode()
         {
             return Value.GetHashCode();
         }
+
+        public readonly string Value;
```
Comment (melkio) [inline src/Transport/ConnectionString.cs]: It's an internal field. Should it be `value` (lowercase)?

File: src/Transport/CreateQueueClients.cs [hunk 3]
```diff
@@ -28,5 +26,7 @@ CloudQueueClient BuildClient(ConnectionString connectionString)
 
             return null;
         }
+
+        ConcurrentDictionary<ConnectionString, CloudQueueClient> destinationQueueClients = new ConcurrentDictionary<ConnectionString, CloudQueueClient>();
```
Comment (melkio) [inline src/Transport/CreateQueueClients.cs]: Since ASQ doesn't support multiaccounts (yet) why are we using a `ConcurrentDictionary`?

**Comment by melkio** [comment]
@danielmarbach a few comments added here and there :)

**Comment by danielmarbach** [comment]
@melkio thanks for the review. The cleanup did not change the code but just move around field and methods according to the style template. So how about we address your points in a dedicated issue? Can you open one?

**Comment by melkio** [comment]
ok @daniel I agree with you on the the third comment. There are two others comments related to "style/cleanup". Are you suggesting to create an issue to rename `Value` field to `value`? Or to inline default values? 
Isn't it better to address them here? Let me know what you prefer...

**Comment by danielmarbach** [comment]
@melkio since you are part of the maintainers I'd just do it as a PR and get it in.

**Pull request merged**